<?xml version="1.0" encoding="UTF-8" standalone="yes" ?>
<test-results name="C:\projetos\Clean-Delphi-API\api_test\exe\api_test.exe" total="1" errors="0" failures="1" ignored="0" inconclusive="0" not-run="0" skipped="0" invalid="0" date="2021-03-09" time="16.137">
  <culture-info current-culture="en" current-uiculture="en" />
  <test-suite type="Assembly" name="api_test.exe" executed="true" result="Failure" success="False" time="16.137" asserts="0">
    <results>
        <test-suite type="Namespace" name="signup_test" executed="true" result="Failure" success="False" time="16.137" asserts="0" >
          <results>
              <test-suite type="Fixture" name="TSignupTest" executed="True" result="Failure" success="False" time="16.137" >
                <results>
                  <test-case name="ShouldReturnError500IfEmailValidatorThrows" executed="True" result="Failure" success="False" time="16.136" asserts="0" >
                    <failure>
                      <message>
                        <![CDATA[ Condition is False when True expected. [Deve retornar {"error":"Internal Server Error"}, valor retornado {"error":"Missing param: passwordConfirmation"}] ]]>
                      </message>
                      <stack-trace>
                        <![CDATA[  ]]>
                      </stack-trace>
                    </failure>
                  </test-case>
                </results>
              </test-suite>
          </results>
        </test-suite>
    </results>
  </test-suite>
</test-results>
